runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -392,4 +427,31 @@ boolean isIgnored(String name)
 			.map(item -> new GameItem(item.getId(), item.getQuantity()))
 			.collect(Collectors.toList());
 	}
+
+	private Collection<LootTrackerRecord> convertToLootTrackerRecord(final Collection<LootRecord> records)
+	{
+		Collection<LootTrackerRecord> trackerRecords = new ArrayList<>();
+		for (LootRecord record : records)
+		{
+			// Convert GameItem drops into LootTrackerItems
+			LootTrackerItem[] drops = record.getDrops().stream().map(itemStack ->
+			{
+				final ItemComposition itemComposition = itemManager.getItemComposition(itemStack.getId());
```

Comment:
Why is all of this logic duplicated from above?

Reply:
No idea, should have separated it into a single function that is used in both places. I'll do so now unless you are working on this.